### PR TITLE
Fixed https://github.com/NuGet/Home/issues/467

### PR DESCRIPTION
**Pull request by danliu**
Clean up resource file for powershell cmdlets.
1. Remove unused ones. The messages are being emitted from other layers. 
2. Unify the naming pattern

### CODE REVIEW

**Comment by danliu** [comment]
@yishaigalatzer @emgarten @deepakaravindr

**Comment by emgarten** [comment]
:shipit:

File: src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Resources.resx [hunk 1]
```diff
@@ -225,69 +201,45 @@
...
-  <data name="FileConflictTitle" xml:space="preserve">
+  <data name="Cmdlets_FileConflictTitle" xml:space="preserve">
     <value>File Conflict</value>
   </data>
-  <data name="PackageNotInstalledInAnyProject" xml:space="preserve">
+  <data name="Cmdlets_PackageNotInstalledInAnyProject" xml:space="preserve">
     <value>'{0}' was not installed in any project. Update failed.</value>
   </data>
-  <data name="UnknownPackage" xml:space="preserve">
+  <data name="Cmdlets_UnknownPackage" xml:space="preserve">
```
Comment (deepakaravindr) [inline src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Resources.resx]: nit: some of the existing ones are prefixed with Cmdlet_

**Comment by deepakaravindr** [comment]
:shipit:

File: src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Resources.resx (hunk 1 above)
Comment (danliu) [inline src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Resources.resx]: good catch! updated.

**Pull request merged**